lua/floating-input.lua

```diff
@@ -56,18 +56,20 @@ function M.input(opts, on_confirm, win_config)
 	-- Enter to confirm
 	vim.keymap.set({ "n", "i", "v" }, "<cr>", function()
 		local lines = vim.api.nvim_buf_get_lines(buffer, 0, 1, false)
-		if lines[1] ~= default and on_confirm then
```

Comment:
It seems that `on_confirm` could be nil for `ui.input`. I guess we need still check it, or assign an empty function for it?

Reply:
Sorry I meant to reply and then I of course forgot :sweat_smile: 

My understanding is that it's mandatory

    {on_confirm}  (`function`) ((input|nil) -> ())

I think only the argument to the function can be nil. Anyway I added a fallback in case it's not passed in the first place as a safety measure